src/python/detectors/module_injection/module_injection.py

```diff
@@ -0,0 +1,20 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+# {fact rule=module-injection@v1.0 defects=1}
+def module_injection_noncompliant():
+    import yaml
+    input_data = input()
+    # Noncompliant: load_all without processing is used.
```

Comment:
Reading this without context, I can't really tell what we mean by `processing` here. I'm also not sure how this describes the `module-injection` issue. Let's discuss offline.